USAGE

```diff
@@ -25,11 +25,17 @@ DESCRIPTION
                                             ├── rev_b0.nii.gz  (optional)
                                             └── t1.nii.gz
 
-    --dti_shells                                Shells selected to compute the dti metrics (generally b < 1200).
-                                                Please write them between quotes e.g. (--dti_shells "0 300").
+    --bids=/path/to/[bids]              BIDS folder following the BIDS convention.
 
-    --fodf_shells                               Shells selected to compute the fodf metrics (generally b > 700).
-                                                Please write them between quotes e.g. (--fodf_shells "0 1000").
+    --bids_config                       BIDS configuration file in json pre-generated by TractoFlow(Read_BIDS process).
```

Comment:
Could we auto-detect the presence of this config file ? Then if its present, fixed and --bids is used, we just load it, no need to change the command lines.

Reply:
I find this suggestion dangerous. I won't go there or maybe latter if it's a need from the community.